ci/tests/arakoon/extend_promote_demote_test.py

```diff
@@ -122,108 +143,74 @@ def teardown():
 
     client = SSHClient(ip)
-    assert client.file_exists(config_file) == True,\
-        "Arakoon config file {0} no longer exists on {1}".format(config_file, ip)
+    assert_true(config_key_exists_on_node(ip, config_file)),\
+        "Arakoon configuration in etcdctl no longer exists on {0}".format(ip)
 
     for directory in [tlog_dir, home_dir, log_dir]:
         assert_false(client.dir_exists(directory)),\
-            "Arakoon directory {0} still exists on {1}".format(config_file, ip)
+            "Arakoon directory {0} still exists on {1}".format(directory, ip)
 
 
 def is_arakoon_dir_config_structure_present(ip, cluster_name, base_dir='/mnt/storage'):
-    config_dir = ArakoonClusterConfig.ARAKOON_CONFIG_DIR.format(cluster_name)
-    config_file = ArakoonClusterConfig.ARAKOON_CONFIG_FILE.format(cluster_name)
+    config_file = ArakoonClusterConfig.ETCD_CONFIG_KEY.format(cluster_name)
     tlog_dir = ArakoonInstaller.ARAKOON_TLOG_DIR.format(base_dir, cluster_name)
     log_dir = ArakoonInstaller.ARAKOON_LOG_DIR.format(cluster_name)
     home_dir = ArakoonInstaller.ARAKOON_HOME_DIR.format(base_dir, cluster_name)
 
     client = SSHClient(ip, username='ovs')
-    assert_true(client.file_exists(config_file)),\
-        "Arakoon config file {0} still exists on {1}".format(config_file, ip)
-    for directory in [config_dir, tlog_dir, home_dir, log_dir]:
+    assert_true(config_key_exists_on_node(ip, config_file)),\
+        "Arakoon configuration in etcdctl doesn't exists on {0}".format(ip)
+    for directory in [tlog_dir, home_dir, log_dir]:
         assert_true(client.dir_exists(directory)),\
-            "Arakoon directory {0} still exists on {1}".format(config_file, ip)
-
-
-def cleanup_arakoon_client_config_files(ips, cluster_name):
-    config_dir = ArakoonClusterConfig.ARAKOON_CONFIG_DIR.format(cluster_name)
-    for ip in ips:
-        client = SSHClient(ip, username='ovs')
-        client.dir_delete(config_dir)
-
-
-def validate_arakoon_config_content(config_file, node_ids):
-    ips = node_ids.keys()
-    ips.sort()
-
-    first_ip = ips[0]
-    client = SSHClient(first_ip, username='root')
-    contents = client.file_read(config_file)
-    cfg = RawConfigParser()
-    cfg.readfp(StringIO(contents))
-
-    logger.info('Arakoon config to validate:\n{0}'.format(str(contents)))
+            "Arakoon directory {0} doesn't exists on {1}".format(directory, ip)
 
-    cluster_id_from_filename = os.path.basename(os.path.splitext(config_file)[0])
-    assert cfg.has_section('global'), 'Arakoon config {0} has no global section'.format(config_file)
-    assert cfg.has_option('global', 'cluster'), 'Arakoon config {0} has no option cluster in global section'.format(config_file)
 
-    cluster = list()
-    for entry in cfg.get('global', 'cluster').split(','):
-        cluster.append(entry)
+def cleanup_arakoon_client_config_files(cluster_name):
+    config_dir = ArakoonClusterConfig.ETCD_CONFIG_KEY.format(cluster_name)[:-7]
```

Comment:
This is very dirty  [:-7]
If we every decide to rename the 'config' part to 'configuration' or whatever, this test will fail
Never do this plz